tensorpipe/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ list(APPEND TP_PUBLIC_HDRS
 
 ### cma
 
-tp_conditional_backend(
+TP_CONDITIONAL_BACKEND(
```

Comment:
So far we've been using this macro lower case, let's keep it consistent (i.e. `tp_conditional_backend()`).

Reply:
@beauby Sorry, that was I who suggested changing it to uppercase, to keep it consistent with the case in Options.cmake: https://github.com/pytorch/tensorpipe/blob/master/cmake/Options.cmake#L13
Would you still like us to change it back to lowercase?